api/classes/class-lightweight-api.php

```diff
@@ -171,32 +172,62 @@ public function set_transient( $name, $value ) {
 	}
 
 	/**
+	 * Delete transient.
+	 *
+	 * @param string $name The transient's name.
+	 */
+	public function delete_transient( $name ) {
+		global $wpdb;
+		$name = '_transient_' . $name;
+
+		wp_cache_delete( $name );
+		$wpdb->delete( // phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery
+			$wpdb->options,
```

Comment:
This deletes the transient from the options table, but as of https://github.com/Automattic/newspack-popups/pull/488 these are in the custom table.